logging/log_converter.cpp

```diff
@@ -0,0 +1,218 @@
+#include <NewRefereeModule.cpp>
+
+#include <google/protobuf/io/zero_copy_stream_impl.h>
+#include <protobuf/LogFrame.pb.h>
+#include <protobuf/referee.pb.h>
+
+#include <QFile>
+#include <QTextStream>
+
+#include <vector>
+
+#include <fcntl.h>
+#include <time.h>
+
+using namespace Packet;
+using namespace std;
+
+#define BAR "|"
+#define MATCH_ID_LENGTH 32
```

Comment:
This is fine as-is, but it's slightly preferred to avoid macros where possible, since they don't obey scope and type rules of c++.  BAR could be `const char` and MATCH_ID_LENGTH could be `const int`.